src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+    
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Move it to class-level like Random.

Reply:
make this object global
`
private ColorSupplier colorSupplier = new ColorSupplier();
`